### DISCUSSION TITLE
Update RKE2

### DISCUSSION THREAD

**Discussion by frit0-rb** [rancher/rke2#5147] [Q&A] [answered]
Good afternoon everyone,

I´m wondering how to update RKE2 in my infra the best way possible.

This is my INFRA:

3 clusters

1 cluster where Rancher is installed with RKE2 installed via Script
1 cluster in origin created via rancher and then decomissioned and now is a Imported cluster
and last one created via Rancher

The actually version of RKE2 is 1.26.7, the last patch is 1.26.11

I want to update all infra.

The main problem I got is when I tried to update first to clusters via Script I got a main problem in the second, because Rancher install the bin an more thing in th path /usr/local/bin but the script install in /usr/bin, so in the case of the second cluster, right now is imported and I cannot update via Rancher so when I run the script to update it like I'm installing a new one RKE2.

How can I proceed to got new patches in this case? Any idea

Thanks

**Comment by brandond** [comment, marked as answer]
What linux distribution are you using? When you provision rke2 via rancher it always uses the tarball to install RKE2, while the install script will default to RPMs on platforms that support it. It sounds like you might need to override the installer to use the tar method, to upgrade your node.

https://github.com/rancher/rke2/blob/c7f5095352c6e3b33be3fde5777c6f4e663dfba7/install.sh#L21-L23

**Reply by frit0-rb** [reply]
Im using Rocky Linux,

I will try your advice.

Thanks

**Comment by frit0-rb** [comment]
thanks @brandond 

Your suggest resolve my problem, thanks for all
